notebooks/notebooks/iclr2022/baseline-attacks.ipynb

```diff
@@ -15,125 +15,405 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        path = directory / pathlib.Path(file)
</pre><p>Nit but if we're going to use <code>pathlib</code> then I think we can make the whole thing <code>pathlib</code>-ified, like:</p><pre class="ql-syntax" spellcheck="false">directory = pathlib.Path("/nas/ucb/ttseng/go_attack/baseline-attack/20221122-rescored")
for path in directory.iterdir():
  if path.suffix != ".sgfs":
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>